packages/flutter/lib/src/material/selection_area.dart

```diff
@@ -63,6 +64,9 @@ class SelectionArea extends StatefulWidget {
   /// If it is null, the platform specific selection control is used.
   final TextSelectionControls? selectionControls;
 
+  /// Called when the selected content changes.
+  final SelectionChangeEventListener? onSelectionChange;
```

Comment:
Will this be called if the user just taps various locations in the text?  I wonder if it's changing an invisible collapsed selection in those cases.

Reply:
This will only be called when an existing selection changes in value. So, if the user just taps on various locations, it would just call this only once when the selection becomes empty, and then it would remain silent until an actual selection is made.